data/templates/default.html5

```diff
@@ -16,6 +16,7 @@ $endif$
   <title>$if(title-prefix)$$title-prefix$ – $endif$$pagetitle$</title>
   <style>
     $styles.html()$
+    $style$
```

Comment:
IS this change intended?

Reply:
yes, see [the MANUAL diff](https://github.com/jgm/pandoc/pull/6601/files#diff-1187c774ee83820d49b1f14f64e3aa4dR2181)

open to discuss all of it :-)

Reply:
ah, but of course this doesn't work as intended, as it's interpreted as markdown... I'll remove it again. The README read:
> `style` can be used to provide additional CSS,
> potentially overriding the default. For example:
> 
> ```
> ---
> title: my doc
> style: |
>   blockquote {
>     border-left: none;
>   }
> ---
> ```